README.md

```diff
@@ -71,6 +71,9 @@ user = intercom.users.create(email: "bob@example.com", name: "Bob Smith", signed
 # Delete a user
 user = intercom.users.find(id: "1")
 deleted_user = intercom.users.delete(user)
+#Send a request to permanently delete a user. User data will be hard deleted within 90 days.
```

Comment:
Can we add this in a separate PR? I'd prefer to keep everything in the same context in this one if that's OK 😄